modules/client-configuration/pages/contact-methods.adoc

```diff
@@ -166,10 +166,10 @@ ssh_push_sudo_user = <user>
 Because clients are in the DMZ and cannot reach the server, you need to use the [command]``mgr-ssh-push-init`` tool to register them with the {productname} Server.
 
 To use the tool, you will need the client hostname or IP address, and the path to a valid bootstrap script on the {productname} Server.
-For more information about bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[].
+For more information about bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[Registering Clients with a Bootstrap Script].
```

Comment:
```suggestion
For more information about bootstrapping, see xref:client-configuration:registration-bootstrap.adoc[].
```

We allow antora to auto-generate the names of the files. It helps prevent breakage.

Reply:
... labels of the references (xrefs)? LAna ist not talking about filenames ;)

Reply:
I see. I am fine reverting this change, it's just the links with '-'s and .adoc at the end don't look so good in the documentation.

Reply:
Yeah, sorry for not being clear! @jcayouette is working on a way to make them render the title of the page instead of the adoc filename in the label, but in the meantime we are settling for that. In the old docs, we had a lot of pages that had label/title mismatches, and it became very difficult to to track down the correct file. This prevents them from drifting apart.